packages/ui/src/components/Switch/base/SwitchBase.tsx

```diff
@@ -16,27 +16,28 @@ export interface SwitchBaseProps {
  * @constructor
  */
 export function SwitchBase(props: SwitchBaseProps) {
-  const {size = 'l', classes, activeColor, inactiveColor} = props;
+  const {size, activeColor, inactiveColor} = props;
   const [checked, setChecked] = useState(false);
   const [active, setActive] = useState(false);
 
   const onChange = useCallback(() => setChecked(state => !state), []);
   const disable = useCallback(() => setActive(false), []);
   const enable = useCallback(() => setActive(true), []);
 
-  const isLarge = size === 'l';
+  const classes = props.classes as (SwitchClasses & Record<string, string>);
+  const s = size?.toUpperCase();
 
-  const rootCn = cn(classes.root, {[classes.rootSizeL]: isLarge});
+  const rootCn = cn(classes.root, classes[`rootSize${s}`]);
   const trackCn = cn(classes.track, {
     [classes.trackChecked]: checked,
-    [classes.trackSizeL]: isLarge,
-    [classes.trackSizeLChecked]: isLarge && checked,
+    [classes[`trackSize${s}`]]: true,
```

Comment:
There is no need to use `true`. We could just move specified class name as follows:

```
const trackCn = cn(classes.track, classes[`trackSize${s}`], { ... });
```